src/css/custom.css

```diff
@@ -241,3 +241,24 @@ td {
     display: block !important;
   }
 }
+
+.tabs {
+  padding-left: 0;
+}
+
+.tabs__item {
+  border-bottom-color: var(--ifm-tabs-color-inactive-border);
+  border-top-left-radius: 0;
```

Comment:
You can simplify these 4 lines into 1 line by using the generic `border-radius`

`border-radius: 0`